src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent-docker.mdx

```diff
@@ -29,7 +29,12 @@ With just a few additions your existing Dockerfile can be used with our Node.js
 
    Install a specific version, or use any of the other options provided by the [`package.json` format](https://docs.npmjs.com/files/package.json#dependencies). Check [the Node.js agent release notes](/docs/release-notes/agent-release-notes/nodejs-release-notes) for information about past agent versions.
 
-2. Depending on how your container is setup, you can edit the `ENTRYPOINT` to include `newrelic` module first with Node.js `-r`/`--require` flag by running `node -r newrelic YOUR_PROGRAM.js`. If you can't control how your program runs, you can load the `newrelic` module before any other module in your program by adding `require('newrelic')`.
+2. Depending on how your container is set up, you can edit the `ENTRYPOINT` to include `newrelic` module first with Node.js `-r`/`--require` flag by running ```node -r newrelic YOUR_PROGRAM.js```
```

Comment:
Can we provide an example console command they can run to do all of this so it's one code block instead of several snippets? It might read something like:

> Depending on your container set-up, you can adjust where the Node.js flag appears. For example:
>
> ```
> node -r newrelic YOUR_PROGRAM.js
> ```

Reply:
I'll rewrite this to note why you'd want to add the -r flag: it's to inject the agent before your application runs. Otherwise, you need to include the agent at the top of your entrypoint file, which is something people often overlook or have trouble with. The agent needs to be loaded before other dependencies run.